build/macos/qt.conf

```diff
@@ -0,0 +1,4 @@
+[Paths]
+Plugins = PlugIns
```

Comment:
`PlugIns` - is this it should be?

Reply:
Yeah, that's the standard name, see [documentation](https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW1)